.github/workflows/ci.yml

```
@@ -25,19 +25,25 @@ jobs:
         tests:
           - name: api-test
             command: /start_tests.sh test_coverage
+            coverage-upload-name: ""
           - name: api-migrations
             command: /start_tests.sh test_migrations
+            coverage-upload-name: ""
           - name: api-lint
             command: /var/lib/awx/venv/awx/bin/tox -e linters
+            coverage-upload-name: ""
           - name: api-swagger
             command: /start_tests.sh swagger
+            coverage-upload-name: ""
           - name: awx-collection
             command: /start_tests.sh test_collection_all
+            coverage-upload-name: "awx-collection"
```

In other places, the strings are unquoted, it'd be nice to follow the surrounding style:
```suggestion
            coverage-upload-name: awx-collection
```